dask/dataframe/rolling.py

```diff
@@ -57,3 +57,78 @@ def rolling(arg, window, *args, **kwargs):
 rolling_quantile = wrap_rolling(pd.rolling_quantile)
 rolling_apply = wrap_rolling(pd.rolling_apply)
 rolling_window = wrap_rolling(pd.rolling_window)
+
+def call_pandas_rolling_method_single(this_partition, rolling_kwargs,
+        method_name, method_args, method_kwargs):
+    # used for the start of the df/series
+    if this_partition.shape[0] < rolling_kwargs['window']:
+        raise NotImplementedError("Window larger than partition size")
+
+    method = getattr(this_partition.rolling(**rolling_kwargs), method_name)
+    return method(*method_args, **method_kwargs)
+
+def call_pandas_rolling_method_with_neighbor(prev_partition, this_partition,
+        rolling_kwargs, method_name, method_args, method_kwargs):
+    # used for everything except for the start
+
+    window = rolling_kwargs['window']
+    if prev_partition.shape[0] < window:
+        raise NotImplementedError("Window larger than partition size")
+
+    if window > 1:
+        extra = window - 1
+        combined = pd.concat([prev_partition.iloc[-extra:], this_partition])
+
+        method = getattr(combined.rolling(window), method_name)
+        applied = method(*method_args, **method_kwargs)
+        return applied.iloc[extra:]
+    else:
+        method = getattr(this_partition.rolling(window), method_name)
+        return method(*method_args, **method_kwargs)
+
+def wrap_rolling_method(name):
+    def inner(self, *args, **kwargs):
+        return self._call_method(name, *args, **kwargs)
+    return inner
+
+class Rolling(object):
```

Comment:
I didn't add one yet. You were going to reflect/research what this should look like, let me know when you've got a chance. @jreback related that there's no real reason for someone to call `Rolling` themselves, so I'm still not sure that `Rolling` is the best place to document things rather than `Series.rolling`/`DataFrame.rolling`. The docstring for pandas' `Rolling` class is mostly about the `__init__` I don't currently mimic -- I can do so in case someone goes searching around if that's what you think is best.